features/merge_articles.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Manage Categories
+  As a blog administrator
+  In order to condense similar articles
+  I want to be able to merge articles
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Creating a new article
+    Given I am on the new article page
```

Comment:
@kariabancroft You are right but this part is not yet complete. During the TDD process at one point it was helpful to see why I test was failing. Thank you for your comment though, I will refactor this.